test/libgit2.jl

```diff
@@ -57,6 +57,14 @@ end
     @test sig3.email == sig.email
 end
 
+@testset "Default config" begin
+    #for now just see that this works
+    cfg = LibGit2.GitConfig()
+    @test isa(cfg, LibGit2.GitConfig)
```

Comment:
Nothing, it was just a "did the thing silently do something insane" sort of test.